core/src/wallet/pool/WalletPool.cpp

```diff
@@ -294,6 +294,12 @@ namespace ledger {
         FuturePtr<AbstractWallet> WalletPool::getWallet(const std::string &name) {
             auto self = shared_from_this();
             return async<std::shared_ptr<AbstractWallet>>([=] () {
+                auto it = self->_wallets.find(WalletDatabaseEntry::createWalletUid(self->getName(), name));
```

Comment:
This is going to conflict with the fix I wrote yesterday and landed this morning.